rotation-service/index.mjs

```diff
@@ -26,47 +26,31 @@ export const handler = async (event) => {
   // We then revoke the API Key that was rolled in the previous invocation.
 
   const item = await itemRequest;
-  const rolledKeySID = getField(item, "rolledKeySID");
+  const rolledKeySID = item.fields.find(f => f.title == "rolledKeySID").value;
   twilioClient.keys(rolledKeySID).remove();
 
   const newAPIKey = await newAPIKeyRequest;
 
-  setField(item, "rolledKeySID", getField(item, "sid"));
-  setField(item, "sid", newAPIKey.sid);
-  setField(item, "secret", newAPIKey.secret);
+  let newItem = {
```

Comment:
**Nit:** could we call this `updatedItem` instead? To make it super clear we're not creating a new item, but updating an existing one